# HG changeset patch
# User Erik Kjellson <erikiiofph7@users.sourceforge.net>
# Date 1387714096 -3600
#      Sun Dec 22 13:08:16 2013 +0100
# Branch stable
# Node ID a8408a424a376c2d0f6f8e76ceb1329ff030bde3
# Parent  442bca7dea44e699730a9606d1d3303bcef9554e
maint: fix spelling of contributor name Erik Kjellson.

diff --git a/doc/interpreter/contributors.in b/doc/interpreter/contributors.in
--- a/doc/interpreter/contributors.in
+++ b/doc/interpreter/contributors.in
@@ -128,17 +128,17 @@ Jarkko Kaleva
 Mohamed Kamoun
 Lute Kamstra
 Fotios Kasolis
 Thomas Kasper
 Joel Keay
 Mumit Khan
 Paul Kienzle
 Aaron A. King
-Erik Kjelsson
+Erik Kjellson
 Arno J. Klaassen
 Alexander Klein
 Geoffrey Knauth
 Heine Kolltveit
 Ken Kouno
 Kacper Kowalik
 Daniel Kraft
 Nir Krakauer
